src/codec_aom.c

```diff
@@ -688,6 +689,9 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
         cfg->g_input_bit_depth = image->depth;
         cfg->g_w = image->width;
         cfg->g_h = image->height;
+        if (disableLaggedOutput) {
+            cfg->g_lag_in_frames = 0;
+        }
```

Comment:
Nit: Move to line 717 or 723 to be near the other code that sets `cfg->g_lag_in_frames`.

Reply:
Done.